packages/embark/src/cmd/dashboard/repl.js

```diff
@@ -18,9 +19,9 @@ class REPL {
     }
   }
 
-  enhancedEval(cmd, context, filename, callback) {
-    this.events.request('console:executeCmd', cmd.trim(), function (err, message) {
-      callback(err, message === undefined ? '' : message); // This way, we don't print undefined
+  enhancedEval(cmd, _context, _filename, _callback) {
+    this.events.request('console:executeCmd', cmd.trim(), (err, message) => {
+      this.logger.info(message === undefined ? '' : message);
```

Comment:
This change causes `undefined` to be printed in the console :/

Reply:
This also does not let the REPL display the results, resulting in a log following by a blank line in the console, without a command prompt. Less than ideal.

Reply:
For objects and for errors, the console will simply display `undefined`

Reply:
Oops, my bad. I had tested with printing objects and it seemed to work. 
The reason I let the logger do the logs for that is so they are propagated to the cockpit too.
Also, that way, we just have to improve the logger. Its support for objects and arrays has always been very limited, so maybe just some kind of detection and JSON.stringify could work